src/components/member/MemberDetailsWizard/WizardForm/WizardForm.jsx

```diff
@@ -152,26 +202,104 @@ class WizardForm extends React.Component {
             onClick={this.onResetFormData}
             type="button"
             value="Reset"
+            disabled={isSubmitDisable}
           />
           <Button
             className="bttn-sbmt button__blue"
             type="submit"
             value="Submit"
+            onClick={this.onClickSubmitButton}
+            disabled={isSubmitDisable}
           />
           <NavigationButton
             currentStep={currentStep}
             totalSteps={totalSteps}
-            onNext={onNext}
-            onPrev={onPrev}
+            onNext={this.onNext}
+            onPrev={this.onPrev}
           />
         </div>
       );
     }
     return <span />;
   };
 
+  setIsSubmitDisable = (isSubmitDisable) => {
+    this.setState({ isSubmitDisable });
+  };
+
+  onChangeFormData = ({ formData }) => {
+    const {
+      formDataConst,
+      formData: prevFormData,
+    } = this.state;
+    const isFormDataEqual = isEqual(formData, prevFormData);
+    if (!isFormDataEqual) {
+      const isFormDataDirty = !isEqual(formData, formDataConst);
+      const isSubmitDisable = !isFormDataDirty;
```

Comment:
No need of storing it in isFormDataDirty variable
```suggestion
       const isSubmitDisabled = !isEqual(updatedFormData, formDataConst);
```

Reply:
done